stratum/hal/lib/tdi/dpdk/dpdk_switch_test.cc

```diff
@@ -61,127 +51,253 @@ MATCHER_P(DerivedFromStatus, status, "") {
 }
 
 constexpr uint64 kNodeId = 13579;
-constexpr int kUnit = 2;
+constexpr int kDevice = 2;
 constexpr char kErrorMsg[] = "Test error message";
 constexpr uint32 kPortId = 2468;
 
-const std::map<uint64, int>& NodeIdToUnitMap() {
-  static auto* map = new std::map<uint64, int>({{kNodeId, kUnit}});
+const absl::flat_hash_map<uint64, int>& NodeIdToDeviceMap() {
+  static auto* map = new absl::flat_hash_map<uint64, int>({{kNodeId, kDevice}});
   return *map;
 }
 
 class DpdkSwitchTest : public ::testing::Test {
  protected:
   void SetUp() override {
-    // Use NiceMock to suppress "uninteresting mock function call" warnings
-    sde_mock_ = absl::make_unique<NiceMock<TdiSdeMock>>();
-    chassis_manager_mock_ = absl::make_unique<NiceMock<DpdkChassisManagerMock>>();
-    node_mock_ = absl::make_unique<NiceMock<TdiNodeMock>>();
-    unit_to_tdi_node_mock_[kUnit] = node_mock_.get();
-    switch_ = DpdkSwitch::CreateInstance(
-        chassis_manager_mock_.get(),
-        sde_mock_.get(),
-        unit_to_tdi_node_mock_);
-#if 0
-    // no 'shutdown'
-    shutdown = false;  // global variable initialization
-#endif
-
-    ON_CALL(*chassis_manager_mock_, GetNodeIdToUnitMap())
-        .WillByDefault(Return(NodeIdToUnitMap()));
+    sde_mock_ = absl::make_unique<TdiSdeMock>();
```

Comment:
Are we getting "uninteresting mock function call" warnings?

I modified this test to use `NiceMock` so we didn't get them.

Reply:
I didn't notice any warnings about uninteresting mock function calls